modules/seqexec/server/src/main/scala/seqexec/server/gpi/Gpi.scala

```diff
@@ -53,6 +54,10 @@ final case class Gpi[F[_]: Sync: Timer](controller: GpiController[F])
   override val observeControl: InstrumentSystem.ObserveControl =
     InstrumentSystem.Uncontrollable
 
+  override def bundleKeywords(
+    ks: List[KeywordBag => SeqActionF[F, KeywordBag]]): SeqActionF[F, KeywordBag] =
+    GdsInstrument.bundleKeywords(ks)
```

Comment:
Ah... never mind. I see the same thing here.
Will there be no instrument-specific keywords, or maybe I am confused.

Reply:
This is happening because I can't put `bundleKeywords` on the `GdsInstrument` trait as it requires as the `F[_]` requires a specifc restriction being `Sync` but can't be done on a trait.

I suppose the best solution would be to eschew inheritance and use a typeclass instead but that's beyond the scope